src/System.Sorting/tests/SortTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Linq;
+using Xunit;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Sorting.Tests
+{
+    public class SortTests
+    {
+        [Fact]
+        public static void TestSort()
```

Comment:
I would break this apart into two methods (one for the string version and one for the int version).  I would also look at using the [Theory] attribute to pass an array of items, so we can remove some of the duplication in these cases (basically you could pull the TestSortHelper code into the Theory's.